raiden/ui/checks.py

```diff
@@ -194,20 +194,4 @@ def check_pfs_configuration(pathfinding_service_address: str) -> None:
 
 
 def check_synced(rpc_client: JSONRPCClient) -> None:
```

Comment:
We could just inline this function, now that only one line is left. It's only called in a single place.